src/Drivers/Behpardakht/Behpardakht.php

```diff
@@ -192,4 +189,65 @@ protected function preparePurchaseData()
             'payerId'           => $payerId
         );
     }
+
+    /**
+     * Convert status to a readable message.
+     *
+     * @param $status
+     *
+     * @return mixed|string
+     */
+    private function translateStatus($status)
+    {
+        $translations = [
+            '0' => 'تراکنش با موفقیت انجام شد',
+            '11' => 'شماره کارت نامعتبر است',
+            '12' => 'موجودی کافی نیست',
+            '13' => 'رمز نادرست است',
+            '14' => 'تعداد دفعات وارد کردن رمز بیش از حد مجاز است',
+            '15' => 'کارت نامعتبر است',
+            '16' => 'دفعات برداشت وجه بیش از حد مجاز است',
+            '17' => 'کاربر از انجام تراکنش منصرف شده است',
+            '18' => 'تاریخ انقضای کارت گذشته است',
+            '19' => 'مبلغ برداشت وجه بیش از حد مجاز است',
+            '111' => 'ضادر کننده کارت نامعتبر است',
+            '112' => 'خطای سوییچ صادر کننده کارت',
+            '113' => 'پاسخی از صادر کننده کارت دریافت نشد',
+            '114' => 'دارنده کارت مجاز به انجام این تراکنش نیست',
+            '21' => 'پذیرنده نامعتبر است',
+            '23' => 'خطای امنیتی رخ داده است',
+            '24' => 'اطلاعات کاربری پذیرنده نامعتبر است',
+            '25' => 'مبلغ نامعتبر است',
+            '31' => 'پاسخ نامعتبر است',
+            '32' => 'فرمت اطلاعات وارد شده صحیح نمی باشد',
+            '33' => 'حساب نامعتبر است',
+            '34' => 'خطای سیستمی',
+            '35' => 'تاریخ نامعتبر است',
+            '41' => 'شماره درخواست تکراری است',
+            '42' => 'تراکنش Sale یافت نشد',
+            '43' => 'قبلا درخواست Verify داده شده است',
+            '44' => 'درخواست Verify یافت نشد',
+            '45' => 'تراکنش Settle شده است',
+            '46' => 'تراکنش Settle نشده است',
+            '47' => 'تراکنش Settle یافت نشد',
+            '48' => 'تراکنش Reverse شده است',
+            '412' => 'شناسه قبض نادرست است',
+            '413' => 'شناسه پرداخت نادرست است',
+            '414' => 'سازمان صادر کننده قبض نامعتبر است',
+            '415' => 'زمان جلسه کاری به پایان رسیده است',
+            '416' => 'خطا در ثبت اطلاعات',
+            '417' => 'شناسه پرداخت کننده نامعتبر است',
+            '418' => 'اشکال در تعریف اطلاعات مشتری',
+            '419' => 'تعداد دفعات ورود اطلاعات از حد مجاز گذشته است',
+            '421' => 'IP نامعتبر است',
+            '51' => 'تراکنش تکراری است',
+            '54' => 'تراکنش مرجع موجود نیست',
+            '55' => 'تراکنش نامعتبر است',
+            '61' => 'خطا در واریز'
```

Comment:
Based on Behpardakht official document _(Ver. 1.03)_
You can find it [here](https://banktest.ir/blog/download/%d8%b1%d8%a7%d9%87%d9%86%d9%85%d8%a7%db%8c-%d8%a7%d8%aa%d8%b5%d8%a7%d9%84-%d8%a8%d9%87-%d8%af%d8%b1%da%af%d8%a7%d9%87-%d9%be%d8%b1%d8%af%d8%a7%d8%ae%d8%aa-%d8%a7%db%8c%d9%86%d8%aa%d8%b1%d9%86%d8%aa/)

Reply:
thanks for your contribution.